pkg/yurthub/gc/gc.go

```diff
@@ -99,7 +100,7 @@ func (m *GCManager) gcPodsWhenRestart() error {
 	cfg := m.restConfigManager.GetRestConfig(true)
 	if cfg == nil {
 		klog.Errorf("could not get rest config, so skip gc pods when restart")
-		return err
+		return errors.New("could not get rest config, so skip gc pods when restart")
```

Comment:
It seems that we do not care about error returned from `gcPodsWhenRestart`. Could we modify this function as
```go
func (m *GCManager) gcPodsWhenRestart()
```

Reply:
+1

Reply:
> It seems that we do not care about error returned from `gcPodsWhenRestart`. Could we modify this function as
> 
> ```go
> func (m *GCManager) gcPodsWhenRestart()
> ```

agree +1